website/blog/2025-12-05-kubernetes-ai-conformance-aks/index.md

```diff
@@ -0,0 +1,188 @@
+
+
+### Security
+
+#### Secure accelerator access
+
+Access to accelerators from within containers must be properly isolated and mediated by the Kubernetes resource management framework (device plugin or DRA) and container runtime, preventing unauthorized access or interference between workloads. See the [AKS secure accelerator access evidence](https://github.com/Azure/AKS/tree/master/ai-conformance/v1.34/secure_accelerator_access) for implementation details.
+
+### Operators
+
+#### AI operator support
+
+Modern AI platforms rely on Kubernetes operators to manage complex resources like training jobs, model servers, and distributed training coordinators. AKS supports the [AI Toolchain Operator (KAITO)](https://learn.microsoft.com/azure/aks/ai-toolchain-operator) and [Ray on AKS](https://learn.microsoft.com/azure/aks/ray-overview), demonstrating that operators with custom resource definitions can be installed and operate correctly.
+
+## How to create an AI-conformant AKS cluster
+
+Getting started with an AI-conformant AKS cluster is straightforward. The key requirement is running a certified Kubernetes version with the appropriate features enabled.
+
+### Step 1. Create an AKS cluster
+
+AKS achieved AI Conformance certification starting with Kubernetes 1.34. Create an AKS cluster running Kubernetes 1.34 or later:
+
+```bash
+az aks create \
+  --resource-group <resource-group> \
+  --name <cluster-name> \
+  --kubernetes-version 1.34.0
+```
+
+### Step 2. Enable the Istio service mesh with Gateway API (optional for advanced routing)
+
+If your AI workloads require advanced traffic management capabilities, you can use the [Istio service mesh add-on with the Gateway API (preview)](https://learn.microsoft.com/azure/aks/istio-gateway-api).
+
+First, install the Azure CLI preview extension and register the feature flag:
+
+```bash
+# Install the aks-preview extension
+az extension add --name aks-preview
+
+# Register the Gateway API feature flag
+az feature register --namespace "Microsoft.ContainerService" --name "ManagedGatewayAPIPreview"
+```
+
+Then enable the Istio service mesh and Gateway API on your cluster:
+
+```bash
+az aks mesh enable \
+  --resource-group <resource-group> \
+  --name <cluster-name>
+
+az aks update \
+  --resource-group <resource-group> \
+  --name <cluster-name> \
+  --enable-gateway-api
+```
+
+### Step 3. Add a GPU node pool
+
+For GPU-accelerated workloads, add a [fully managed GPU node pool (preview)](https://learn.microsoft.com/azure/aks/aks-managed-gpu-nodes) with [GPU-enabled VMs](https://learn.microsoft.com/azure/virtual-machines/sizes/overview#gpu-accelerated). AKS automatically installs the GPU driver, device plugin, and DCGM metrics exporter.

```

Comment:
Do we want to callout this is for Nvidia GPU driver, device plugin, and DCGM metrics exporter?